### DISCUSSION TITLE
Problems with the way with use data and GIS

### DISCUSSION THREAD

**Discussion by ctran1400** [albertkun/21S-ASIAAM-191A#37] [Past Thinking Caps]
Issues that appear with data and the use of data usually stems from the human biases. For example, in one of my previous classes we studied a case study about the LAPD seeking assistance from a GIS lab in helping to lay out the geographic locations of Muslim populations groups around LA. The LAPD stated it was to take a look into their history, culture, economic status, etc and to identify which communities with in the larger Muslim community could be susceptible to violet extremism and terrorism. We discussed in class how this data collection could be greatly unethical because not only does it potentially perpetuate harmful stereotypes, the definition of a community being "susceptible" to extremism could be easily twisted to fit anything by the LAPD. Overall, data collection like that if it would to occur could result in neighborhoods being violated by excessive policing simply because the region is dominated by one race, religion, or culture.

**Comment by KenNL42** [comment]
I agree with bias in data can lead to a misleading conclusion. Sometimes the data may show a correlation between two features, but it does not always tell the whole picture. Like how you use your anecdote, terrorism and crime rate may be high in Muslim community, but maybe the crime rate per household is low when comparing to other community. When using data, it is important to be aware of bias in data which could lead to incorrect result or hatred toward a group of people.

**Comment by jsxu315** [comment]
I agree with the points you made! It's important to not just consider a project from a purely technical sense; we also need to consider why we're building it in the first place. It made me look up predictive policing algorithms; while it's cool to have an algorithm that can 'predict' certain events for convenience, we should step back and consider why we're trying to predict people's (in particular, minority groups') behavior in the first place. I came across [this article](https://www.technologyreview.com/2020/07/17/1005396/predictive-policing-algorithms-racist-dismantled-machine-learning-bias-criminal-justice/) about predictive policing, which has this quote from Dorothy Roberts that explains why people see the concept of predictive policing as fundamentally flawed: “Racism has always been about predicting, about making certain racial groups seem as if they are predisposed to do bad things and therefore justify controlling them.”
The article also mentions how a NYPD commissioner wanted to increase the department's use of predictive policing technologies after the 2020 protests — it's really unsettling to think about how this misuse of technology can put people at greater risk while hiding prejudices and decreasing accountability for cops.

**Reply by albertkun** [reply]
@jsxu315 What should be even more appalling about predictive policing for us is that a UCLA professor, Reed Hutchinson, helped to develop it and made tons of money from selling this terrible tool. The algorithm he uses was one that was actually used for predicting earthquakes.. This is really an example of why we have to leverage our skills to fight back as these systems really want to capitalize on the targets that have been historically marginalized/discriminated against.
